### PR TITLE
sunxi-tools: (new)

### PR DESCRIPTION
**Pull request by elitak**
This is my first submission to nixpkgs, so please afford these changes some extra scrutiny.

I have tested each build manually.

Some concerns:
- Do I need to make myself the maintainer somehow?
- Am I submitting this to the correct branch?
- Have I versioned default.nix and 1.3.nix correctly here? (they are both sunxi-tools-1.3, but the former uses a newer git revision).

### CODE REVIEW

File: pkgs/development/tools/sunxi-tools/1.2.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    make all misc
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp bin2fex bootinfo fel fex2bin fexc nand-part phoenix_info pio $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Tools for Allwinner A10 devices.";
```
Comment (jagajaga) [inline pkgs/development/tools/sunxi-tools/1.2.nix]: Please remove dot at the end of the description.

File: pkgs/development/tools/sunxi-tools/1.3.nix [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    make all misc
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp bin2fex fex2bin phoenix_info sunxi-bootinfo sunxi-fel sunxi-fexc sunxi-nand-part sunxi-pio $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Tools for Allwinner A10 devices.";
```
Comment (jagajaga) [inline pkgs/development/tools/sunxi-tools/1.3.nix]: Here too.

File: pkgs/development/tools/sunxi-tools/default.nix [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    make all misc
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp bin2fex fex2bin phoenix_info sunxi-bootinfo sunxi-fel sunxi-fexc sunxi-nand-part sunxi-pio $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Tools for Allwinner A10 devices.";
```
Comment (jagajaga) [inline pkgs/development/tools/sunxi-tools/default.nix]: And here.

**Comment by jagajaga** [comment]
Is there a reason we need so many versions of this pkg?

**Comment by jagajaga** [comment]
Yes, if you want to, you can become a maintainer.

**Comment by elitak** [comment]
There's probably no need for older versions, so I've removed them. I wasn't aware fewer versions was preferred and was including them for completeness.

I've added myself as the maintainer.

**Comment by jagajaga** [comment]
Please squash commits into one.

**Comment by elitak** [comment]
Okay, done.

**Comment by elitak** [comment]
I also took the liberty of fixing an out-of-lexographical-order alias in maintainers.nix.

**Comment by jagajaga** [comment]
Please rename commit to `sunxi-tools: init at 1.3`

**Comment by elitak** [comment]
Done.

**Pull request merged**